fields-material/build.gradle.kts

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2022 IceRock MAG Inc. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+plugins {
+    id("dev.icerock.moko.gradle.multiplatform.mobile")
+    id("dev.icerock.moko.gradle.detekt")
+    id("dev.icerock.moko.gradle.publication")
+    id("dev.icerock.moko.gradle.stub.javadoc")
+}
+
+android {
+    buildFeatures.viewBinding = true
+}
+
+dependencies {
+    api(projects.fieldsCore)
+    api(projects.fieldsFlow)
+    api(projects.fieldsLivedata)
```

Comment:
we should not depend on flow or livedata. we should depend only on core.
with your implementation in project will be both - flow and livedata implementations, that not needed